tests/playwright/specs/404.spec.js

```diff
@@ -25,7 +25,7 @@ test.describe(
             const goBackLink = page.getByTestId('link-go-back');
             await expect(goBackLink).toBeVisible();
             await goBackLink.click();
-            await page.waitForURL('**/?automation=true', {
+            await page.waitForURL('**/?automation=true*', {
```

Comment:
If there's no rush would this be an actual improvement while there:

```suggestion
            await page.waitForURL('**/en-US/?automation=true*', {
```

to not pass literally "any" page, incl. 404s and others?

Reply:
Actually, this is enough here - have tested locally against Staging
```suggestion
            await page.waitForURL('/en-US/?automation=true*', {
```

Reply:
But let's go with your suggestion, np